lib/helpers/auditor.js

```diff
@@ -13,7 +13,8 @@ module.exports = audit;
  * @returns {undefined} Does not return anything.
  */
 function audit(err, req, res) {
-    req.headers = req._headers;
+    req.headers = req.getHeaders();
```

Comment:
is `req` an instance of [http.ClientRequest](https://nodejs.org/docs/latest-v14.x/api/http.html#http_class_http_clientrequest)?

Reply:
Yes

Reply:
Do we add the `.getHeaders()` method? It clearly works because tests passed but I can't find it on the Node.js docs...

Reply:
We don't. I'll have to double check, it's been a while since I worked on this PR 😅

Reply:
so ClientRequest extends OutgoingMessage, which has getHeaders. Might be just a documentation error on Node.js, which says it extends Stream.

Reply:
And it's been like that since at least 2013 🤷🏻‍♀️

Reply:
Anyway, the deprecation warning is explicit about OutgoingMessage behavior, not ClientRequest, and it recommends using getHeaders: https://nodejs.org/dist/latest-v14.x/docs/api/deprecations.html#deprecations_dep0066_outgoingmessage_prototype_headers_outgoingmessage_prototype_headernames

Reply:
oh...